docs/develop/zzz/application/charge_plan.md

```
@@ -10,29 +10,29 @@
 - **跳过计划**：当前计划因电量不足或次数限制无法继续时，继续尝试后续计划。
 - **每日重置**：按游戏刷新日每天首次运行前清零所有计划的已运行次数。
 - **恢复电量**：体力不足时按配置使用储蓄电量、以太电池或两者。
-- **双倍活动**：开启后会在识别电量后检查实战模拟室双倍次数；命中活动时按配置的实战模拟室任务生成运行期临时计划，优先执行但不写入正式计划列表。
+- **双倍活动**：开启后会在识别资源栏后检查实战模拟室双倍次数；命中活动时按配置的实战模拟室任务生成运行期临时计划，优先执行但不写入正式计划列表。
 
 ## 主要流程
 
 ### 1. 初始化阶段
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)